### PR TITLE
Update link to licenses

### PR DESCRIPTION
**Pull request by Philippus**
looks like the footnote anchor is gone

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ If your project does not use a license, you may opt out of specifying one:
 bintrayOmitLicense := true
 ```
 ##### public (default)
-If your project uses a license, Bintray supports those listed [here](https://bintray.com/docs/api/#_footnote_1). If you are new to software licenses you may
+If your project uses a license, Bintray supports those listed on the bottom of the [Bintray API documentation](https://bintray.com/docs/api/). If you are new to software licenses you may
```
Comment (dwijnand) [inline README.md]: We can use https://bintray.com/docs/api/#_licenses

Comment (Philippus) [reply README.md]: or better: https://bintray.com/docs/api/#_get_oss_licenses ?

Comment (dwijnand) [reply README.md]: Sure!

Comment (Philippus) [reply README.md]: I've changed it.

**Pull request merged**